superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilters.test.ts

```diff
@@ -328,7 +328,7 @@ describe('Horizontal FilterBar', () => {
     openVerticalFilterBar();
     setFilterBarOrientation('horizontal');
 
-    enterNativeFilterEditModal(false);
```

Comment:
Reverting logic from https://github.com/apache/superset/pull/23314 now there the two `/api/v1/dataset/{pk}` GET requests are different.